### DISCUSSION TITLE
How to get Joint Sensor

### DISCUSSION THREAD

**Discussion by JKevin9** [isaac-sim/IsaacLab#1456] [Q&A]
Hi everyone!  I've got a teensy-weensy problem with getting the joint torque sensor. I'm not mentioning applied_torque. But here's the thing: torque_wanted = applied_torque + external_torque. Is this possible in IsaacLab? 🤔 If anyone can help me out, I will be truly, deeply grateful! 😊🙏

**Comment by JKevin9** [comment]
`asset._root_physx_view.get_dof_projected_joint_forces()` What is this? Is this what I want?

**Comment by dxyy1** [comment]
I am not too sure about any standalone joint torque sensor class, but I think the class `omni.isaac.lab.assets.articulation` deals with joints; to get joint torques, you can access the data container `omni.isaac.lab.assets.articulation.articulation_data` class

Not sure if this is what you are looking for though.
